src/lib/__tests__/BoundedCache-test.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-unused-expressions */
 import BoundedCache from '../BoundedCache';
 
-const sandbox = sinon.sandbox.create();
+const sandbox = sinon.createSandbox();
```

Comment:
I tried to reduce the usage of sinon, but I wasn't able to fully excise it, mainly due to some complicated `withArgs` usage.

Reply:
How is sinon being used in this test?